tensorboard/webapp/metrics/views/metrics_container.scss

```diff
@@ -24,6 +24,11 @@ nav {
   border-right: 1px solid mat-color($tb-foreground, border);
   flex: none;
   width: 340px;
+
+  @include tb-dark-theme {
+    background-color: map-get($tb-dark-background, background);
+    border-right-color: mat-color($tb-dark-foreground, border);
+  }
```

Comment:
To check my understanding, is this equivalent to
```
nav {
  @include tb-theme-background-prop(background-color, background);
  @include tb-theme-foreground-prop(border-right, border, 1px solid);
```
?

Reply:
Correct; I still cannot make up my mind on which pattern is better when there are more than N *-props call but this is the pattern that I wanted to include in the code base for reference.